app/experimenter/experiments/api/v5/types.py

```diff
@@ -129,10 +130,17 @@ class NimbusExperimentTargetingConfigType(graphene.ObjectType):
 class NimbusFeatureConfigType(DjangoObjectType):
     id = graphene.Int()
     application = NimbusExperimentApplicationEnum()
+    sets_prefs = graphene.Boolean()
 
     class Meta:
         model = NimbusFeatureConfig
 
+    def resolve_sets_prefs(root, info):
+        if feature := Features.find(root.application, root.slug):
+            return bool(feature.sets_prefs)
```

Comment:
Is this not already a bool?  Or you mean it could be `None`?

Reply:
`feature.set_prefs` is a list of strings, which we don't want to return becuase graphql is expecting bool